WcaOnRails/db/structure.sql

```diff
@@ -18,19 +18,19 @@ CREATE TABLE `Competitions` (
   `cityName` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
   `countryId` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
   `information` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
-  `year` smallint(5) unsigned NOT NULL DEFAULT '0',
```

Comment:
The structure.sql shouldn't be updated with this database migration and is likely pulling in local configs from your database. It'd be best to remove this file from the commit (and not commit it upon further changes when the structure is not updated).
`git checkout master -- WcaOnRails/db/structure.sql`

Reply:
I'll revert this